LICENSE.txt

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
```

Comment:
```suggestion
     // You should remove the below lines too.
```

Reply:
Thanks for that! I've removed the appendix bit now!